packages/golden_toolkit/lib/src/testing_tools.dart

```diff
@@ -281,8 +281,8 @@ Future<void> legacyPrimeAssets(WidgetTester tester) async {
 /// See also:
 /// * [GoldenToolkitConfiguration.primeAssets] to configure a global asset prime function.
 Future<void> defaultPrimeAssets(WidgetTester tester) async {
-  final imageElements = find.byType(Image).evaluate();
-  final containerElements = find.byType(Container).evaluate();
+  final imageElements = find.byType(Image, skipOffstage: false).evaluate();
+  final containerElements = find.byType(DecoratedBox, skipOffstage: false).evaluate();
```

Comment:
DecoratedBox is ultimately what we care about... Container is only one particular consumer.

skipOffstage will find an image widgets in the widget tree even if they aren't currently on the stage at the time waitForAssets() is called.